app/javascript/widget/api/contacts.js

```diff
@@ -12,9 +12,14 @@ export default {
       ...userObject,
     });
   },
-  setCustomAttibutes(customAttributes = {}) {
+  setCustomAttributes(customAttributes = {}) {
     return API.patch(buildUrl('widget/contact'), {
       custom_attributes: customAttributes,
     });
   },
+  deleteCustomAttributes(customAttributes = {}) {
+    return API.post(buildUrl('widget/contact/destroy_custom_attributes'), {
+      custom_attributes: customAttributes,
```

Comment:
If we are not sending multiple attributes at once, then we should rename it to custom_attribute or something else.

Reply:
We can pass array of strings